OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -109,6 +109,21 @@ public void onPageScrollStateChanged(int state) {
         mNextBtn.setOnClickListener(view -> navigateNextComponent());
     }
 
+    @Override
+    public void onSaveInstanceState(Bundle outState) {
+        /*
+         * If the youtube player is not in a proper state than it throws the  IllegalStateException.
```

Comment:
2 nits:
- then instead of than.
- remove extra space before `IllegalStateException`.

Reply:
done.